light-client-verifier/src/operations/voting_power.rs

```diff
@@ -87,6 +87,19 @@ pub trait VotingPowerCalculator: Send + Sync {
         }
     }
 
+    /// Check if there is 2/3rd overlap between an untrusted header and untrusted validator set
+    fn check_signers_overlap(
+        &self,
+        untrusted_header: &SignedHeader,
+        untrusted_validators: &ValidatorSet,
+    ) -> Result<(), VerificationError> {
+        self.check_signers_overlap_level(
```

Comment:
I guess this should be a call to the `check_enough_signers_overlap` method above...

Reply:
01c5112f3625905bb00db13c030254532410adf4